lore/developers/dev-environment/personal-cloud-accounts.md

```diff
@@ -17,7 +17,7 @@ The BonnyCI project offers personal cloud accounts to core developers. This allo
 
 ## Obtaining an account
 
-Accounts can be requested by creating a pull request to the [Hoist repo](https://github.com/BonnyCI/hoist/), specifically the [create-users-projects.yml](https://github.com/BonnyCI/hoist/blob/master/create-users-projects.yml) playbook. Simply add your details; desired username, email address for contact, and whether or not you'll want `cloud_admin` rights. Once the pull request has been approved, an existing developer with `cloud_admin` rights will execute the playbook to make your account live. A temporary password will be used, which needs to be changed immediately.
+Accounts can be requested by creating a pull request to the [Hoist repo](https://github.com/BonnyCI/hoist/), specifically the [create-users-projects.yml](https://github.com/BonnyCI/hoist/blob/master/create-cloud-resources.yml) playbook. Simply add your details; desired username, email address for contact, and whether or not you'll want `cloud_admin` rights. Once the pull request has been approved, an existing developer with `cloud_admin` rights will execute the playbook to make your account live. A temporary password will be used, which needs to be changed immediately.
```

Comment:
Also, while making this change, why don't we change the line to use breaks at 80 characters?